src/schema/v2/OrderedSet/addOrderedSetItemMutation.ts

```diff
@@ -86,7 +91,7 @@ export const addOrderedSetItemMutation = mutationWithClientMutationId<
 
       const { item_type } = await setLoader(id)
```

Comment:
since you're fetching the set here, could inject this response into the `return` below.

``` ts
const set = awaitSetLoader(id)
return { item_type: set.item_type, set, ...res }
```

and then you don't need a fetch or resolver in the success response type:

``` ts
set: {
  type: OrderedSetType,
  // resolve: ({ setId }, _options, { setLoader }) => setLoader(setId), -> don't need this
},
```

Saves you one fetch of the set. No biggie tho either way

Reply:
Nice. Good catch. Added in [5fe58e0](https://github.com/artsy/metaphysics/pull/5012/commits/5fe58e04d7e34d23310653f42085699feeddbeef).